core/resources/sdf.fs

```diff
@@ -11,25 +11,42 @@ uniform LOWP vec3 u_color;
 varying vec2 v_uv;
 varying float v_alpha;
 
-float minOutlineD = 0.25;
-float maxOutlineD = 0.35;
-float minInsideD = 0.38;
-float maxInsideD = 0.55;
-float mixFactor = 0.7;
-const vec3 outlineColor = vec3(0.7);
-
 const float gamma = 2.2;
+const float tint = 1.8;
+const float sdf = 0.8;
 
-void main(void) {
-    float distance = texture2D(u_tex, v_uv).a;
-
-    float a1 = smoothstep(minInsideD, maxInsideD, distance);
-    float a2 = smoothstep(minOutlineD, maxOutlineD, distance);
+float contour(in float d, in float w, in float off) {
+    return smoothstep(off - w, off + w, d);
+}
 
-    a1 = pow(a1, 1.0 / gamma);
-    a2 = pow(a2, 1.0 / gamma);
+float sample(in vec2 uv, float w, in float off) {
+    return contour(texture2D(u_tex, uv).a, w, off);
+}
 
-    float d = clamp(distance / (1.0 - distance), 0.0, 1.0);
+float sampleAlpha(in vec2 uv, float distance, in float off) {
+    float dscale = 0.354; // 1 / sqrt(2)
+    vec2 duv = dscale * (dFdx(uv) + dFdy(uv));
+    vec4 box = vec4(uv - duv, uv + duv);
+    float alpha = contour(distance, distance, off);
+    
+    float asum = sample(box.xy, distance, off)
+               + sample(box.zw, distance, off)
+               + sample(box.xw, distance, off)
+               + sample(box.zy, distance, off);
```

Comment:
The issue I brought up yesterday regarding texture coordinates modified in the shader is evidently called "dependent texture reads" and it's explained [here](https://developer.apple.com/library/ios/documentation/3DDrawing/Conceptual/OpenGLES_ProgrammingGuide/BestPracticesforShaders/BestPracticesforShaders.html) in the section labeled "Be Aware of Dynamic Texture Lookups"